### DISCUSSION TITLE
Help me create a world record!

### DISCUSSION THREAD

**Discussion by KrlitosForever** [community/community#120174] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello to the entire community, my name is Carlos and this afternoon I said I want to demonstrate that this is the best community that exists on the planet, and I thought, How do I do it? and I said, easy, we'll make the book with the largest number of authors and nationalities possible. That's why I turn to you so that together we can create this book. The repository is https://github.com/KrlitosForever/Community-in-Guinness-World-Record I hope you will be part of this madness. Who knows, maybe we will achieve the world record

**Comment by ebndev** [comment]
Hi @KrlitosForever, we’re glad you’re here and appreciate your enthusiasm! 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
